rules-tests/TypeDeclaration/Rector/ClassMethod/StrictArrayParamDimFetchRector/Fixture/skip_unknown_parent.php.inc

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\StrictArrayParamDimFetchRector\Fixture;
+
+use Rector\Tests\TypeDeclaration\Rector\ClassMethod\StrictArrayParamDimFetchRector\Source\SomeInterface;
+
+class SkipUnknownParent extends UnknownParentClass
+{
+    public function go($item)
```

Comment:
if we don't know the parent class (because e.g. its not autoloadable), which should be pessimistic and do no changes